doc/releases/changelog-dev.md

```diff
@@ -225,67 +227,80 @@
 
-* A `Sum` symbolic class is added that allows users to represent the sum of operators.
+* Many convenient additions have been made regarding Operator arithmetic.
   [(#2475)](https://github.com/PennyLaneAI/pennylane/pull/2475)
+  [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
+  [(#2807)](https://github.com/PennyLaneAI/pennylane/pull/2807)
+  [(#2849)](https://github.com/PennyLaneAI/pennylane/pull/2849)
+  
+When we refer to Operator "arithmetic" we mean operations like addition, 
+multiplication and exponentiation. Quality-of-life features have been added in 
+each of these categories:
 
-  The `Sum` class provides functionality like any other PennyLane operator. We can
-  get the matrix, eigenvalues, terms, diagonalizing gates and more.
+  - Linear combinations:
+
+  Linearly combine any Operators to create a new Operator whose matrix, terms, 
+  and eigenvalues can be accessed as per usual:
 
   ```pycon
-  >>> summed_op = qml.op_sum(qml.PauliX(0), qml.PauliZ(0))
+  >>> summed_op = qml.PauliX(0) - 5*qml.PauliZ(0)
   >>> summed_op
-  PauliX(wires=[0]) + PauliZ(wires=[0])
+  PauliX(wires=[0]) - 5 * PauliZ(wires=[0])
   >>> qml.matrix(summed_op)
-  array([[ 1,  1],
-         [ 1, -1]])
+  array([[ -5,  1],
+         [ 1, 5]])
   >>> summed_op.terms()
-  ([1.0, 1.0], (PauliX(wires=[0]), PauliZ(wires=[0])))
+  ([1.0, -5.0], (PauliX(wires=[0]), PauliZ(wires=[0])))
   ```
 
-  The `summed_op` can also be used inside a `qnode` as an observable.
-  If the circuit is parameterized, then we can also differentiate through the
-  sum observable.
+  You can also add scalars to Operators, where the interpretation is that the 
+  scalar is a properly-sized identity matrix. 
+
+  ```pycon
+  >>> sum_op = 5 + qml.PauliX(0)
+  >>> sum_op.matrix()
+  array([[5., 1.],
+         [1., 5.]])
+  ```
+
+  Linearly-combined Operators can also be used inside of a `qnode` as an observable 
+  while maintaining differentiability.
 
   ```python
   sum_op = Sum(qml.PauliX(0), qml.PauliZ(1))
   dev = qml.device("default.qubit", wires=2)
 
-  @qml.qnode(dev, grad_method="best")
+  @qml.qnode(dev)
   def circuit(weights):
-        qml.RX(weights[0], wires=0)
-        qml.RY(weights[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.RX(weights[2], wires=1)
-        return qml.expval(sum_op)
+      qml.RX(weights[0], wires=0)
+      qml.RY(weights[1], wires=1)
+      qml.CNOT(wires=[0, 1])
+      qml.RX(weights[2], wires=1)
+      return qml.expval(sum_op)
   ```
 
   ```pycon
   >>> weights = qnp.array([0.1, 0.2, 0.3], requires_grad=True)
   >>> qml.grad(circuit)(weights)
   tensor([-0.09347337, -0.18884787, -0.28818254], requires_grad=True)
   ```
-
-* Added `__add__` and `__pow__` dunder methods to the `qml.operation.Operator` class so that users can combine operators
-  more naturally. [(#2807)](https://github.com/PennyLaneAI/pennylane/pull/2807)
+* Added `__add__` and `__pow__` dunder methods to the `qml.operation.Operator` class 
```

Comment:
I'm a little confused here. Indeed, 

```pycon
>>> summed_op = qml.RX(phi=1.23, wires=0) + qml.RZ(phi=3.14, wires=0)
RX(1.23, wires=[0]) + RZ(3.14, wires=[0])
```

and 

```pycon
>>> summed_op = qml.RX(phi=1.23, wires=0) + qml.RZ(phi=3.14, wires=0)
RX(1.23, wires=[0]) + RZ(3.14, wires=[0])
```

But,

```pycon
>>> summed_op = qml.PauliX(0) + qml.PauliZ(0)
 (1) [X0]
+ (1) [Z0]
```

and 

```pycon
>>> summed_op = qml.op_sum(qml.PauliX(0), qml.PauliZ(0))
PauliX(wires=[0]) + PauliZ(wires=[0])
```

Reply:
So, what is the function of `qml.op_sum` relative to the `+` operator?

Reply:
@albi3ro Could you give us the reasoning behind the different outputs here? Thanks in advance :)

Reply:
`qml.op_sum` is a direct way to create a sum of *any* number of operators.  For example, you could do:
```
qml.op_sum(qml.RX(1.0, 0), qml.PauliZ("a"), qml.CNOT((0,1)) )
```
You could also do:
```
qml.RX(1.0, 0) + qml.PauliZ("a") + qml.CNOT((0,1))
```
but that would create a slightly different, more nested Operator that represents the same thing.

Right now, `Operator.__sum__` uses `qml.op_sum`, but `Observable.__sum__` creates a `Hamiltonian`, consistent with the previous behavior of summing `Observable`'s.
So doing:
```
qml.PauliX(0) + qml.PauliY(0)
```
would return a `Hamiltonian`, but 
```
qml.RX(1,0) + qml.RY(1,0)
```
would return a `Sum`.

Reply:
Thanks Christina! 💯